### ISSUE TITLE
ACC-255.1 (Level A) _ Modify Modules _ Icon _ No text alternative

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3134]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	Icon
Affected Check Point:	ACC-255.1
Short text:	ACC-255.1 (Level A) _ Modify Modules _ Icon _ No text alternative

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Mouse hover the Side navigation , ‘Feedback’ and ‘Information’ buttons across the application.

3. Issue Type: Manual
Observed Behavior:
There is no text alternative provided for Side navigation , ‘Feedback’ and ‘Information’ buttons across the application.
Screenshot:
 
Case 2:
 
Case 3:
 
Case 4:
 

Expected Behavior:
An appropriate and meaningful text alternative must be provided for Side navigation , ‘Feedback’ and ‘Information’ buttons across the application which serves the purpose of the UI.
Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha
